pkgs/applications/virtualization/tart/default.nix

```
@@ -10,11 +10,17 @@
 }:
 stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "tart";
-  version = "2.4.3";
+  version = "2.6.0";
 
-  src = fetchurl {
-    url = "https://github.com/cirruslabs/tart/releases/download/${finalAttrs.version}/tart.tar.gz";
-    sha256 = "sha256-cXisvF+W/Uxe3Q0ZRhkvF13UWXxbsIQSzG172lzwruo=";
+  src = let
+    binaryTarget = "arm64";
+    throwError = throw "Unknown target ${binaryTarget} for tart package!";
+    hash = {
+        "arm64" = "sha256-QtVqgmjYpLAhFibW1DIVWNYiOjHB1X/YY6zRVB+1soA=";
+    }.${binaryTarget} or throwError;
+  in fetchurl {
+      url = "https://github.com/cirruslabs/tart/releases/download/${finalAttrs.version}/tart-${binaryTarget}.tar.gz";
+      inherit hash;
```

Since tart is arm64-specific software (right? in any case we're not supporting any other platforms at the moment), I don't think we need this abstraction and can merely add arm64 into the URL:
```suggestion
  src = fetchurl {
      url = "https://github.com/cirruslabs/tart/releases/download/${finalAttrs.version}/tart-arm64.tar.gz";
      hash = "sha256-QtVqgmjYpLAhFibW1DIVWNYiOjHB1X/YY6zRVB+1soA=";
```